src/edu/csus/ecs/pc2/exports/ccs/ResultsFile.java

```diff
@@ -96,16 +96,12 @@ private int getMedian(StandingsRecord[] srArray) {
      * @return
      * @throws IllegalContestState
```

Comment:
The above line should be removed, due to the change below.

Reply:
throw removed.